src/js/components/DataTable/DataTable.js

```diff
@@ -490,6 +503,13 @@ const DataTable = ({
               setRowExpand={setRowExpand}
             />
           )}
+          {placeholder && (!items || items.length === 0) && (
+            <tbody>
```

Comment:
Will this result in two `tbody` elements in the DOM, since Body is also inserting one?

Reply:
This work in progress does, but I was going to eliminate that if you think this idea is worth it. The extra empty tbody didn't really cause any visual issues but I wouldn't leave it there

Reply:
:) Yup. It's basically columns.length + 1 for onSelect I think